cmd/res/configuration.toml

```diff
@@ -1,5 +1,12 @@
 [Writable]
 LogLevel = 'INFO'
+  # Example InsecureSecrets configuration that simulates SecretStore for when EDGEX_SECURITY_SECRET_STORE=false
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.Sample]
+     path = "sample"
+      [Writable.InsecureSecrets.Sample.Secrets]
+        username = ""
+        password = ""
```

Comment:
Note that this can be eliminated since the service doesn't have any secrets. We have device-mqtt as an example of how these are used for services that have secrets.